shashi/practiceTest2.js

```diff
@@ -0,0 +1,67 @@
+//calculate Factorial of a number
+const calcFactorial = num => {
+  if(num < 0){
+    return `Not a value number to calculate factorial`;
+  } 
+  if (num === 0 || num === 1){
+    return 1;
```

Comment:
calcFactorial(0) should not return 1

Reply:
Thats what it says when I google it :) factorial of 0 is 1.